content/blog/2024-06-19-gsoc-environment-variables.mdx

```diff
@@ -0,0 +1,62 @@
+---
+title: "GSoC'24: Supporting User-provided, Long-lived Environmental, Variables for Unikraft Builds"
+description: |
+  The goal of this project is to improve Unikraft's
+  tooling—more especially, the Kraft command-line tool—so that it can dynamically inject
+  user-supplied variables into the build system. 
+publishedDate: 2024-06-19
+tags:
+- GSoC'24   
+- Unikernals
+- Unikraft
+- Golang
+authors:
+- Ujjwal Mahar
+---
+
+## GSoC'24: Supporting User-provided, Long-lived Environmental, Variables for Unikraft Builds
+
+<img width="100px" src="https://summerofcode.withgoogle.com/assets/media/gsoc-generic-badge.svg" align="right" />
+
+
+### Project Overview
+
+Unikraft is a highly adaptable cloud-based library operating
+system with a high degree of modularity. However, developers' ability to add custom variables to the build process is hampered
+by the rigidity of the current tooling. The goal of this project is to improve Unikraft's
+tooling—more especially, the Kraft command-line tool—so that it can dynamically inject
+user-supplied variables into the build system. Features like changing GCC versions and
+compile-time settings are included in this. Developers can customize Unikraft builds to meet
+specific needs by providing dynamic configuration, enhancing flexibility.
```

Comment:
Newlines on new sentences only please.

```suggestion
Unikraft is a highly adaptable cloud-based library operating system with a high degree of modularity.
However, developers' ability to add custom variables to the build process is hampered by the rigidity of the current tooling.
The goal of this project is to improve Unikraft's tooling—more especially, the Kraft command-line tool—so that it can dynamically inject user-supplied variables into the build system.
Features like changing GCC versions and compile-time settings are included in this.
Developers can customize Unikraft builds to meet specific needs by providing dynamic configuration, enhancing flexibility.
```

Reply:
Same for other cases